### PR TITLE
Handle deleted instance/cluster when refreshing db/user

### PR DESCRIPTION
**Pull request by schirevko**

### CODE REVIEW

**Review by ftersin — APPROVED** [review]

File: mcs/resource_mcs_db_database.go [hunk 1]
```diff
@@ -164,6 +163,18 @@ func resourceDatabaseDatabaseRead(d *schema.ResourceData, meta interface{}) erro
...
 
+	if dbmsType == dbmsTypeInstance {
+		_, err := instanceGet(DatabaseV1Client, dbmsID).extract()
+		if err != nil {
+			return checkDeleted(d, err, "Error retrieving mcs_db_database")
+		}
+	} else {
+		_, err := dbClusterGet(DatabaseV1Client, dbmsID).extract()
+		if err != nil {
+			return checkDeleted(d, err, "Error retrieving mcs_db_database")
```
Comment (irlndts) [inline mcs/resource_mcs_db_database.go:174]: you can use one `if err != nil` for both cases after if-else block, let's move it outside please

File: mcs/resource_mcs_db_user.go [hunk 2]
```diff
@@ -179,6 +179,18 @@ func resourceDatabaseUserRead(d *schema.ResourceData, meta interface{}) error {
 		dbmsType = dbmsTypeInstance
 	}
 
+	if dbmsType == dbmsTypeInstance {
+		_, err := instanceGet(DatabaseV1Client, dbmsID).extract()
+		if err != nil {
```
Comment (irlndts) [inline mcs/resource_mcs_db_user.go:184]: and here

File: mcs/resource_mcs_db_database.go (hunk 1 above)
Comment (schirevko) [reply mcs/resource_mcs_db_database.go]: done

File: mcs/resource_mcs_db_user.go (hunk 2 above)
Comment (schirevko) [reply mcs/resource_mcs_db_user.go]: done

**Review by irlndts — APPROVED** [review]

**Review by sti-jans — APPROVED** [review]

**Pull request merged**